lua/telescope/actions/init.lua

```diff
@@ -143,6 +144,50 @@ function actions.toggle_selection(prompt_bufnr)
   current_picker:toggle_selection(current_picker:get_selection_row())
 end
 
+--- Multi select all entries.
+--- - Note: selected entries may include results not visible in the results popup.
+---@param prompt_bufnr number: The prompt bufnr
+function actions.select_all(prompt_bufnr)
+  local current_picker = action_state.get_current_picker(prompt_bufnr)
+  -- indices are 1-indexed, rows are 0-indexed
```

Comment:
this comment is out of place here. `utils.map_entries` should mention it in its header.

Reply:
Ok